docs/en/operations/named-collections.md

```diff
@@ -443,3 +443,59 @@ SELECT dictGet('dict', 'b', 1);
 │ a                       │
 └─────────────────────────┘
 ```
+
+## Named collections for accessing Kafka
+
+The description of parameters see [Kafka](../engines/table-engines/integrations/kafka.md).
+
+### DDL example
```

Comment:
Here we should probably have an example of the creation of a named collection instead of `CREATE TABLE queue` query

Reply:
Hi @evillique, 

You are right.
Added.

Cheers